dotCMS/src/integration-test/java/com/dotcms/enterprise/publishing/remote/PushPublishBundleGeneratorTest.java

```diff
@@ -597,5 +605,80 @@ public void Test_GenerateBundle_PageWithTemplateDeleted_PageSuccessfullyAddedToB
 
     }
 
+    /**
+     * Given Scenario: If a file asset has zero files and CONTENT_ALLOW_ZERO_LENGTH_FILES set to true,
+     *                  we should be able to generate the bundle without issues.
+     * ExpectedResult: The bundle should be generated without issues and the page added to the bundle.
+     */
+    @Test
+    public void Test_GenerateBundle_FileAssetZeroBytes_ContentAllowZeroLengthFilesSetToTrue_Success() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5701517)